packages/flutter/test/widgets/list_view_relayout_test.dart

```diff
@@ -124,9 +124,6 @@ void main() {
     expect(find.text('300'), findsOneWidget);
     expect(find.text('400'), findsOneWidget);
     expect(find.text('100'), findsNothing);
-
-    final RenderSliverList list = tester.renderObject(find.byType(SliverList));
-    expect(list.geometry.scrollExtent, equals(700.0));
```

Comment:
The estimated scroll extent didn't exactly match the extent of the two visible widgets anymore because the child manager (now) knows that we've underflowed.